2021-01-23 21:06:08 INFO run.py:219 GNS3 server version 2.2.17
2021-01-23 21:06:08 INFO run.py:221 Copyright (c) 2007-2021 GNS3 Technologies Inc.
2021-01-23 21:06:08 INFO run.py:224 Config file /home/saad/.config/GNS3/2.2/gns3_server.conf loaded
2021-01-23 21:06:08 WARNING run.py:230 Local mode is enabled. Beware, clients will have full control on your filesystem
2021-01-23 21:06:08 INFO run.py:243 Running with Python 3.9.1 and has PID 55288
2021-01-23 21:06:08 INFO run.py:79 Current locale is en_US.UTF-8
2021-01-23 21:06:08 INFO web_server.py:318 Starting server on localhost:3080
2021-01-23 21:06:08 INFO __init__.py:62 Load controller configuration file /home/saad/.config/GNS3/2.2/gns3_controller.conf
2021-01-23 21:06:08 INFO __init__.py:66 Controller is starting
2021-01-23 21:06:08 INFO compute.py:64 Create compute local
2021-01-23 21:06:08 INFO compute.py:364 Connecting to compute 'local'
2021-01-23 21:06:08 INFO web_log.py:233 ::1 [23/Jan/2021:16:06:08 +0000] "GET /v2/compute/capabilities HTTP/1.1" 200 552 "-" "Python/3.9 aiohttp/3.6.2"
2021-01-23 21:06:08 INFO notification_handler.py:50 New client has connected to compute WebSocket
2021-01-23 21:06:08 INFO compute.py:434 Connected to compute 'local' WebSocket 'http://localhost:3080/v2/compute/notifications/ws'
2021-01-23 21:06:08 INFO web_log.py:233 ::1 [23/Jan/2021:16:06:08 +0000] "GET /v2/version HTTP/1.1" 200 227 "-" "GNS3 QT Client v2.2.17"
2021-01-23 21:06:08 INFO web_log.py:233 ::1 [23/Jan/2021:16:06:08 +0000] "GET /v2/templates HTTP/1.1" 200 8595 "-" "GNS3 QT Client v2.2.17"
2021-01-23 21:06:08 INFO notification_handler.py:73 New client has connected to controller WebSocket
2021-01-23 21:06:08 INFO web_log.py:233 ::1 [23/Jan/2021:16:06:08 +0000] "GET /v2/computes HTTP/1.1" 200 1023 "-" "GNS3 QT Client v2.2.17"
2021-01-23 21:06:08 INFO web_log.py:233 ::1 [23/Jan/2021:16:06:08 +0000] "GET /v2/appliances?symbol_theme=Affinity-square-red HTTP/1.1" 200 741681 "-" "GNS3 QT Client v2.2.17"
2021-01-23 21:06:08 INFO web_log.py:233 ::1 [23/Jan/2021:16:06:08 +0000] "GET /v2/projects HTTP/1.1" 200 1464 "-" "GNS3 QT Client v2.2.17"
2021-01-23 21:06:08 INFO web_log.py:233 ::1 [23/Jan/2021:16:06:08 +0000] "GET /v2/version HTTP/1.1" 200 227 "-" "GNS3 QT Client v2.2.17"
2021-01-23 21:06:08 INFO web_log.py:233 ::1 [23/Jan/2021:16:06:08 +0000] "GET /v2/projects HTTP/1.1" 200 1464 "-" "GNS3 QT Client v2.2.17"
2021-01-23 21:06:11 INFO project.py:78 Project eca8779b-e255-4fd3-8992-7e3ae5413625 with path '/home/saad/GNS3/projects/network automation' created
2021-01-23 21:06:11 INFO web_log.py:233 ::1 [23/Jan/2021:16:06:11 +0000] "POST /v2/compute/projects HTTP/1.1" 201 313 "-" "Python/3.9 aiohttp/3.6.2"
2021-01-23 21:06:11 INFO qemu_vm.py:201 QEMU VM "S1" [9e868b82-c2e7-4ac7-a8ff-95ecf0a71548] has set the QEMU path to /bin/qemu-system-x86_64
2021-01-23 21:06:11 INFO qemu_vm.py:603 QEMU VM "S1" [9e868b82-c2e7-4ac7-a8ff-95ecf0a71548]: MAC address changed to 0c:36:25:15:48:00
2021-01-23 21:06:11 INFO qemu_vm.py:551 QEMU VM "S1" [9e868b82-c2e7-4ac7-a8ff-95ecf0a71548]: number of Ethernet adapters changed to 1
2021-01-23 21:06:11 INFO qemu_vm.py:128 QEMU VM "S1" [9e868b82-c2e7-4ac7-a8ff-95ecf0a71548] has been created
2021-01-23 21:06:11 INFO base_node.py:282 Qemu: S1 [9e868b82-c2e7-4ac7-a8ff-95ecf0a71548] created
2021-01-23 21:06:11 INFO qemu_vm.py:551 QEMU VM "S1" [9e868b82-c2e7-4ac7-a8ff-95ecf0a71548]: number of Ethernet adapters changed to 8
2021-01-23 21:06:11 INFO qemu_vm.py:244 QEMU VM "S1" [9e868b82-c2e7-4ac7-a8ff-95ecf0a71548] has set the QEMU hda_disk_image path to /home/saad/GNS3/images/QEMU/vios_l2-adventerprisek9-m.03.2017.qcow2
2021-01-23 21:06:11 INFO qemu_vm.py:603 QEMU VM "S1" [9e868b82-c2e7-4ac7-a8ff-95ecf0a71548]: MAC address changed to 0c:36:25:93:82:00
2021-01-23 21:06:11 INFO qemu_vm.py:720 QEMU VM "S1" [9e868b82-c2e7-4ac7-a8ff-95ecf0a71548] has set the process priority to normal
2021-01-23 21:06:11 INFO qemu_vm.py:743 QEMU VM "S1" [9e868b82-c2e7-4ac7-a8ff-95ecf0a71548] has set the RAM to 768
2021-01-23 21:06:11 INFO web_log.py:233 127.0.0.1 [23/Jan/2021:16:06:11 +0000] "POST /v2/compute/projects/eca8779b-e255-4fd3-8992-7e3ae5413625/qemu/nodes HTTP/1.1" 201 1684 "-" "Python/3.9 aiohttp/3.6.2"
2021-01-23 21:06:11 INFO qemu_vm.py:201 QEMU VM "R1" [1b0875e1-3164-4107-b7fc-8d38acb55ee7] has set the QEMU path to /usr/sbin/qemu-system-i386
2021-01-23 21:06:11 INFO qemu_vm.py:603 QEMU VM "R1" [1b0875e1-3164-4107-b7fc-8d38acb55ee7]: MAC address changed to 0c:36:25:5e:e7:00
2021-01-23 21:06:11 INFO qemu_vm.py:551 QEMU VM "R1" [1b0875e1-3164-4107-b7fc-8d38acb55ee7]: number of Ethernet adapters changed to 1
2021-01-23 21:06:11 INFO qemu_vm.py:128 QEMU VM "R1" [1b0875e1-3164-4107-b7fc-8d38acb55ee7] has been created
2021-01-23 21:06:11 INFO base_node.py:282 Qemu: R1 [1b0875e1-3164-4107-b7fc-8d38acb55ee7] created
2021-01-23 21:06:11 INFO qemu_vm.py:551 QEMU VM "R1" [1b0875e1-3164-4107-b7fc-8d38acb55ee7]: number of Ethernet adapters changed to 8
2021-01-23 21:06:11 INFO qemu_vm.py:244 QEMU VM "R1" [1b0875e1-3164-4107-b7fc-8d38acb55ee7] has set the QEMU hda_disk_image path to /home/saad/GNS3/images/QEMU/vios-adventerprisek9-m.vmdk.SPA.156-2.T
2021-01-23 21:06:11 INFO qemu_vm.py:348 QEMU VM "R1" [1b0875e1-3164-4107-b7fc-8d38acb55ee7] has set the QEMU hda disk interface to virtio
2021-01-23 21:06:11 INFO qemu_vm.py:244 QEMU VM "R1" [1b0875e1-3164-4107-b7fc-8d38acb55ee7] has set the QEMU hdb_disk_image path to /home/saad/GNS3/images/QEMU/IOSv_startup_config.img
2021-01-23 21:06:11 INFO qemu_vm.py:371 QEMU VM "R1" [1b0875e1-3164-4107-b7fc-8d38acb55ee7] has set the QEMU hdb disk interface to virtio
2021-01-23 21:06:11 INFO qemu_vm.py:603 QEMU VM "R1" [1b0875e1-3164-4107-b7fc-8d38acb55ee7]: MAC address changed to 0c:36:25:76:89:00
2021-01-23 21:06:11 INFO qemu_vm.py:720 QEMU VM "R1" [1b0875e1-3164-4107-b7fc-8d38acb55ee7] has set the process priority to normal
2021-01-23 21:06:11 INFO qemu_vm.py:743 QEMU VM "R1" [1b0875e1-3164-4107-b7fc-8d38acb55ee7] has set the RAM to 512
2021-01-23 21:06:11 INFO web_log.py:233 ::1 [23/Jan/2021:16:06:11 +0000] "POST /v2/compute/projects/eca8779b-e255-4fd3-8992-7e3ae5413625/qemu/nodes HTTP/1.1" 201 1836 "-" "Python/3.9 aiohttp/3.6.2"
2021-01-23 21:06:11 INFO nat.py:51 NAT node 'NAT1' configured to use NAT interface 'virbr0'
2021-01-23 21:06:11 INFO base_node.py:655 Starting new uBridge hypervisor localhost:34455
2021-01-23 21:06:11 INFO hypervisor.py:169 starting ubridge: ['/usr/bin/ubridge', '-H', 'localhost:34455']
2021-01-23 21:06:11 INFO hypervisor.py:171 logging to /home/saad/GNS3/projects/network automation/project-files/builtin/fc8b1192-f9df-4d09-ace8-297b139f04b6/ubridge.log
2021-01-23 21:06:11 INFO hypervisor.py:179 ubridge started PID=55305
2021-01-23 21:06:11 INFO base_node.py:658 Hypervisor localhost:34455 has successfully started
2021-01-23 21:06:11 INFO ubridge_hypervisor.py:83 Connected to uBridge hypervisor on localhost:34455 after 0.1031 seconds
2021-01-23 21:06:11 INFO cloud.py:215 Cloud "NAT1" [fc8b1192-f9df-4d09-ace8-297b139f04b6] has been created
2021-01-23 21:06:11 INFO web_log.py:233 127.0.0.1 [23/Jan/2021:16:06:11 +0000] "POST /v2/compute/projects/eca8779b-e255-4fd3-8992-7e3ae5413625/nat/nodes HTTP/1.1" 201 553 "-" "Python/3.9 aiohttp/3.6.2"
2021-01-23 21:06:11 INFO docker_vm.py:1058 Docker container "ubuntu-1" [910bf8f4-0655-495c-91de-87034fb5bc33]: number of Ethernet adapters changed to 1
2021-01-23 21:06:11 INFO docker_vm.py:406 Docker container 'ubuntu-1' [910bf8f4-0655-495c-91de-87034fb5bc33] created
2021-01-23 21:06:11 INFO web_log.py:233 ::1 [23/Jan/2021:16:06:11 +0000] "POST /v2/compute/projects/eca8779b-e255-4fd3-8992-7e3ae5413625/docker/nodes HTTP/1.1" 201 949 "-" "Python/3.9 aiohttp/3.6.2"
2021-01-23 21:06:11 INFO web_log.py:233 127.0.0.1 [23/Jan/2021:16:06:11 +0000] "POST /v2/compute/projects/eca8779b-e255-4fd3-8992-7e3ae5413625/ports/udp HTTP/1.1" 201 243 "-" "Python/3.9 aiohttp/3.6.2"
2021-01-23 21:06:11 INFO web_log.py:233 ::1 [23/Jan/2021:16:06:11 +0000] "POST /v2/compute/projects/eca8779b-e255-4fd3-8992-7e3ae5413625/ports/udp HTTP/1.1" 201 243 "-" "Python/3.9 aiohttp/3.6.2"
2021-01-23 21:06:11 INFO qemu_vm.py:1346 QEMU VM "S1" [9e868b82-c2e7-4ac7-a8ff-95ecf0a71548]: NIO UDP added to adapter 0
2021-01-23 21:06:11 INFO web_log.py:233 127.0.0.1 [23/Jan/2021:16:06:11 +0000] "POST /v2/compute/projects/eca8779b-e255-4fd3-8992-7e3ae5413625/qemu/nodes/9e868b82-c2e7-4ac7-a8ff-95ecf0a71548/adapters/0/ports/0/nio HTTP/1.1" 201 378 "-" "Python/3.9 aiohttp/3.6.2"
2021-01-23 21:06:11 INFO cloud.py:396 Cloud "NAT1" [fc8b1192-f9df-4d09-ace8-297b139f04b6]: NIO NIO UDP bound to port 0
2021-01-23 21:06:11 INFO web_log.py:233 ::1 [23/Jan/2021:16:06:11 +0000] "POST /v2/compute/projects/eca8779b-e255-4fd3-8992-7e3ae5413625/nat/nodes/fc8b1192-f9df-4d09-ace8-297b139f04b6/adapters/0/ports/0/nio HTTP/1.1" 201 377 "-" "Python/3.9 aiohttp/3.6.2"
2021-01-23 21:06:11 INFO web_log.py:233 127.0.0.1 [23/Jan/2021:16:06:11 +0000] "POST /v2/compute/projects/eca8779b-e255-4fd3-8992-7e3ae5413625/ports/udp HTTP/1.1" 201 243 "-" "Python/3.9 aiohttp/3.6.2"
2021-01-23 21:06:11 INFO web_log.py:233 ::1 [23/Jan/2021:16:06:11 +0000] "POST /v2/compute/projects/eca8779b-e255-4fd3-8992-7e3ae5413625/ports/udp HTTP/1.1" 201 243 "-" "Python/3.9 aiohttp/3.6.2"
2021-01-23 21:06:11 INFO qemu_vm.py:1346 QEMU VM "S1" [9e868b82-c2e7-4ac7-a8ff-95ecf0a71548]: NIO UDP added to adapter 1
2021-01-23 21:06:11 INFO web_log.py:233 127.0.0.1 [23/Jan/2021:16:06:11 +0000] "POST /v2/compute/projects/eca8779b-e255-4fd3-8992-7e3ae5413625/qemu/nodes/9e868b82-c2e7-4ac7-a8ff-95ecf0a71548/adapters/1/ports/0/nio HTTP/1.1" 201 378 "-" "Python/3.9 aiohttp/3.6.2"
2021-01-23 21:06:11 INFO qemu_vm.py:1346 QEMU VM "R1" [1b0875e1-3164-4107-b7fc-8d38acb55ee7]: NIO UDP added to adapter 1
2021-01-23 21:06:11 INFO web_log.py:233 ::1 [23/Jan/2021:16:06:11 +0000] "POST /v2/compute/projects/eca8779b-e255-4fd3-8992-7e3ae5413625/qemu/nodes/1b0875e1-3164-4107-b7fc-8d38acb55ee7/adapters/1/ports/0/nio HTTP/1.1" 201 378 "-" "Python/3.9 aiohttp/3.6.2"
2021-01-23 21:06:11 INFO web_log.py:233 127.0.0.1 [23/Jan/2021:16:06:11 +0000] "POST /v2/compute/projects/eca8779b-e255-4fd3-8992-7e3ae5413625/ports/udp HTTP/1.1" 201 243 "-" "Python/3.9 aiohttp/3.6.2"
2021-01-23 21:06:11 INFO web_log.py:233 ::1 [23/Jan/2021:16:06:11 +0000] "POST /v2/compute/projects/eca8779b-e255-4fd3-8992-7e3ae5413625/ports/udp HTTP/1.1" 201 243 "-" "Python/3.9 aiohttp/3.6.2"
2021-01-23 21:06:11 INFO docker_vm.py:960 Docker container 'ubuntu-1' [910bf8f4-0655-495c-91de-87034fb5bc33]: NIO UDP added to adapter 0
2021-01-23 21:06:11 INFO web_log.py:233 127.0.0.1 [23/Jan/2021:16:06:11 +0000] "POST /v2/compute/projects/eca8779b-e255-4fd3-8992-7e3ae5413625/docker/nodes/910bf8f4-0655-495c-91de-87034fb5bc33/adapters/0/ports/0/nio HTTP/1.1" 201 380 "-" "Python/3.9 aiohttp/3.6.2"
2021-01-23 21:06:11 INFO qemu_vm.py:1346 QEMU VM "S1" [9e868b82-c2e7-4ac7-a8ff-95ecf0a71548]: NIO UDP added to adapter 2
2021-01-23 21:06:11 INFO web_log.py:233 ::1 [23/Jan/2021:16:06:11 +0000] "POST /v2/compute/projects/eca8779b-e255-4fd3-8992-7e3ae5413625/qemu/nodes/9e868b82-c2e7-4ac7-a8ff-95ecf0a71548/adapters/2/ports/0/nio HTTP/1.1" 201 378 "-" "Python/3.9 aiohttp/3.6.2"
2021-01-23 21:06:11 INFO web_log.py:233 ::1 [23/Jan/2021:16:06:10 +0000] "POST /v2/projects/eca8779b-e255-4fd3-8992-7e3ae5413625/open HTTP/1.1" 201 771 "-" "GNS3 QT Client v2.2.17"
2021-01-23 21:06:11 INFO project_handler.py:271 New client has connected to the notification stream for project ID 'eca8779b-e255-4fd3-8992-7e3ae5413625' (WebSocket method)
2021-01-23 21:06:11 INFO web_log.py:233 ::1 [23/Jan/2021:16:06:11 +0000] "GET /v2/projects/eca8779b-e255-4fd3-8992-7e3ae5413625/nodes HTTP/1.1" 200 15808 "-" "GNS3 QT Client v2.2.17"
2021-01-23 21:06:11 INFO web_log.py:233 ::1 [23/Jan/2021:16:06:11 +0000] "GET /v2/projects/eca8779b-e255-4fd3-8992-7e3ae5413625/links HTTP/1.1" 200 4122 "-" "GNS3 QT Client v2.2.17"
2021-01-23 21:06:11 INFO web_log.py:233 ::1 [23/Jan/2021:16:06:11 +0000] "GET /v2/projects/eca8779b-e255-4fd3-8992-7e3ae5413625/drawings HTTP/1.1" 200 205 "-" "GNS3 QT Client v2.2.17"
2021-01-23 21:06:14 INFO web_log.py:233 127.0.0.1 [23/Jan/2021:16:06:14 +0000] "POST /v2/compute/projects/eca8779b-e255-4fd3-8992-7e3ae5413625/nat/nodes/fc8b1192-f9df-4d09-ace8-297b139f04b6/start HTTP/1.1" 204 244 "-" "Python/3.9 aiohttp/3.6.2"
2021-01-23 21:06:15 INFO qemu_vm.py:1621 logging to /home/saad/GNS3/projects/network automation/project-files/qemu/9e868b82-c2e7-4ac7-a8ff-95ecf0a71548/qemu-img.log
2021-01-23 21:06:15 INFO qemu_vm.py:1623 Executing qemu-img with: /bin/qemu-img check /home/saad/GNS3/images/QEMU/vios_l2-adventerprisek9-m.03.2017.qcow2
2021-01-23 21:06:15 INFO qemu_vm.py:1621 logging to /home/saad/GNS3/projects/network automation/project-files/qemu/1b0875e1-3164-4107-b7fc-8d38acb55ee7/qemu-img.log
2021-01-23 21:06:15 INFO qemu_vm.py:1623 Executing qemu-img with: /usr/sbin/qemu-img check /home/saad/GNS3/images/QEMU/vios-adventerprisek9-m.vmdk.SPA.156-2.T
2021-01-23 21:06:15 INFO qemu_vm.py:1627 /bin/qemu-img returned with 0
2021-01-23 21:06:15 INFO base_node.py:655 Starting new uBridge hypervisor localhost:46125
2021-01-23 21:06:15 INFO qemu_vm.py:1627 /usr/sbin/qemu-img returned with 0
2021-01-23 21:06:15 INFO base_node.py:822 Qemu: 'S1' [9e868b82-c2e7-4ac7-a8ff-95ecf0a71548]:local UDP tunnel created between port 10006 and 10007
2021-01-23 21:06:15 INFO base_node.py:822 Qemu: 'S1' [9e868b82-c2e7-4ac7-a8ff-95ecf0a71548]:local UDP tunnel created between port 10008 and 10009
2021-01-23 21:06:15 INFO base_node.py:822 Qemu: 'S1' [9e868b82-c2e7-4ac7-a8ff-95ecf0a71548]:local UDP tunnel created between port 10010 and 10011
2021-01-23 21:06:15 INFO base_node.py:822 Qemu: 'S1' [9e868b82-c2e7-4ac7-a8ff-95ecf0a71548]:local UDP tunnel created between port 10012 and 10013
2021-01-23 21:06:15 INFO base_node.py:822 Qemu: 'S1' [9e868b82-c2e7-4ac7-a8ff-95ecf0a71548]:local UDP tunnel created between port 10014 and 10015
2021-01-23 21:06:15 INFO base_node.py:822 Qemu: 'S1' [9e868b82-c2e7-4ac7-a8ff-95ecf0a71548]:local UDP tunnel created between port 10016 and 10017
2021-01-23 21:06:15 INFO base_node.py:822 Qemu: 'S1' [9e868b82-c2e7-4ac7-a8ff-95ecf0a71548]:local UDP tunnel created between port 10018 and 10019
2021-01-23 21:06:15 INFO base_node.py:822 Qemu: 'S1' [9e868b82-c2e7-4ac7-a8ff-95ecf0a71548]:local UDP tunnel created between port 10020 and 10021
2021-01-23 21:06:15 INFO hypervisor.py:169 starting ubridge: ['/usr/bin/ubridge', '-H', 'localhost:46125']
2021-01-23 21:06:15 INFO hypervisor.py:171 logging to /home/saad/GNS3/projects/network automation/project-files/docker/910bf8f4-0655-495c-91de-87034fb5bc33/ubridge.log
2021-01-23 21:06:15 INFO hypervisor.py:179 ubridge started PID=55448
2021-01-23 21:06:15 INFO base_node.py:658 Hypervisor localhost:46125 has successfully started
2021-01-23 21:06:15 INFO qemu_vm.py:1621 logging to /home/saad/GNS3/projects/network automation/project-files/qemu/1b0875e1-3164-4107-b7fc-8d38acb55ee7/qemu-img.log
2021-01-23 21:06:15 INFO qemu_vm.py:1623 Executing qemu-img with: /usr/sbin/qemu-img check /home/saad/GNS3/images/QEMU/IOSv_startup_config.img
2021-01-23 21:06:15 INFO qemu_vm.py:1627 /usr/sbin/qemu-img returned with 63
2021-01-23 21:06:15 INFO qemu_vm.py:1011 Starting QEMU with: /bin/qemu-system-x86_64 -name S1 -m 768M -smp cpus=1,sockets=1 -enable-kvm -machine smm=off -boot order=c -drive 'file=/home/saad/GNS3/projects/network automation/project-files/qemu/9e868b82-c2e7-4ac7-a8ff-95ecf0a71548/hda_disk.qcow2,if=ide,index=0,media=disk,id=drive0' -uuid 9e868b82-c2e7-4ac7-a8ff-95ecf0a71548 -serial telnet:127.0.0.1:5001,server,nowait -monitor tcp:127.0.0.1:46611,server,nowait -net none -device e1000,mac=0c:36:25:93:82:00,netdev=gns3-0 -netdev socket,id=gns3-0,udp=127.0.0.1:10007,localaddr=127.0.0.1:10006 -device e1000,mac=0c:36:25:93:82:01,netdev=gns3-1 -netdev socket,id=gns3-1,udp=127.0.0.1:10009,localaddr=127.0.0.1:10008 -device e1000,mac=0c:36:25:93:82:02,netdev=gns3-2 -netdev socket,id=gns3-2,udp=127.0.0.1:10011,localaddr=127.0.0.1:10010 -device e1000,mac=0c:36:25:93:82:03,netdev=gns3-3 -netdev socket,id=gns3-3,udp=127.0.0.1:10013,localaddr=127.0.0.1:10012 -device e1000,mac=0c:36:25:93:82:04,netdev=gns3-4 -netdev socket,id=gns3-4,udp=127.0.0.1:10015,localaddr=127.0.0.1:10014 -device e1000,mac=0c:36:25:93:82:05,netdev=gns3-5 -netdev socket,id=gns3-5,udp=127.0.0.1:10017,localaddr=127.0.0.1:10016 -device e1000,mac=0c:36:25:93:82:06,netdev=gns3-6 -netdev socket,id=gns3-6,udp=127.0.0.1:10019,localaddr=127.0.0.1:10018 -device e1000,mac=0c:36:25:93:82:07,netdev=gns3-7 -netdev socket,id=gns3-7,udp=127.0.0.1:10021,localaddr=127.0.0.1:10020 -display none
2021-01-23 21:06:15 INFO qemu_vm.py:1013 logging to /home/saad/GNS3/projects/network automation/project-files/qemu/9e868b82-c2e7-4ac7-a8ff-95ecf0a71548/qemu.log
2021-01-23 21:06:15 INFO qemu_vm.py:1021 QEMU VM "S1" started PID=55473
2021-01-23 21:06:15 INFO base_node.py:655 Starting new uBridge hypervisor localhost:41995
2021-01-23 21:06:15 INFO base_node.py:822 Qemu: 'R1' [1b0875e1-3164-4107-b7fc-8d38acb55ee7]:local UDP tunnel created between port 10022 and 10023
2021-01-23 21:06:15 INFO base_node.py:822 Qemu: 'R1' [1b0875e1-3164-4107-b7fc-8d38acb55ee7]:local UDP tunnel created between port 10024 and 10025
2021-01-23 21:06:15 INFO base_node.py:822 Qemu: 'R1' [1b0875e1-3164-4107-b7fc-8d38acb55ee7]:local UDP tunnel created between port 10026 and 10027
2021-01-23 21:06:15 INFO base_node.py:822 Qemu: 'R1' [1b0875e1-3164-4107-b7fc-8d38acb55ee7]:local UDP tunnel created between port 10028 and 10029
2021-01-23 21:06:15 INFO base_node.py:822 Qemu: 'R1' [1b0875e1-3164-4107-b7fc-8d38acb55ee7]:local UDP tunnel created between port 10030 and 10031
2021-01-23 21:06:15 INFO base_node.py:822 Qemu: 'R1' [1b0875e1-3164-4107-b7fc-8d38acb55ee7]:local UDP tunnel created between port 10032 and 10033
2021-01-23 21:06:15 INFO base_node.py:822 Qemu: 'R1' [1b0875e1-3164-4107-b7fc-8d38acb55ee7]:local UDP tunnel created between port 10034 and 10035
2021-01-23 21:06:15 INFO base_node.py:822 Qemu: 'R1' [1b0875e1-3164-4107-b7fc-8d38acb55ee7]:local UDP tunnel created between port 10036 and 10037
2021-01-23 21:06:15 INFO hypervisor.py:169 starting ubridge: ['/usr/bin/ubridge', '-H', 'localhost:41995']
2021-01-23 21:06:15 INFO hypervisor.py:171 logging to /home/saad/GNS3/projects/network automation/project-files/qemu/9e868b82-c2e7-4ac7-a8ff-95ecf0a71548/ubridge.log
2021-01-23 21:06:15 INFO hypervisor.py:179 ubridge started PID=55494
2021-01-23 21:06:15 INFO base_node.py:658 Hypervisor localhost:41995 has successfully started
2021-01-23 21:06:15 INFO qemu_vm.py:1011 Starting QEMU with: /usr/sbin/qemu-system-i386 -name R1 -m 512M -smp cpus=1,sockets=1 -enable-kvm -boot order=c -drive 'file=/home/saad/GNS3/projects/network automation/project-files/qemu/1b0875e1-3164-4107-b7fc-8d38acb55ee7/hda_disk.qcow2,if=virtio,index=0,media=disk,id=drive0' -drive 'file=/home/saad/GNS3/projects/network automation/project-files/qemu/1b0875e1-3164-4107-b7fc-8d38acb55ee7/hdb_disk.qcow2,if=virtio,index=1,media=disk,id=drive1' -uuid 1b0875e1-3164-4107-b7fc-8d38acb55ee7 -serial telnet:127.0.0.1:5002,server,nowait -monitor tcp:127.0.0.1:44821,server,nowait -net none -device e1000,mac=0c:36:25:76:89:00,netdev=gns3-0 -netdev socket,id=gns3-0,udp=127.0.0.1:10023,localaddr=127.0.0.1:10022 -device e1000,mac=0c:36:25:76:89:01,netdev=gns3-1 -netdev socket,id=gns3-1,udp=127.0.0.1:10025,localaddr=127.0.0.1:10024 -device e1000,mac=0c:36:25:76:89:02,netdev=gns3-2 -netdev socket,id=gns3-2,udp=127.0.0.1:10027,localaddr=127.0.0.1:10026 -device e1000,mac=0c:36:25:76:89:03,netdev=gns3-3 -netdev socket,id=gns3-3,udp=127.0.0.1:10029,localaddr=127.0.0.1:10028 -device e1000,mac=0c:36:25:76:89:04,netdev=gns3-4 -netdev socket,id=gns3-4,udp=127.0.0.1:10031,localaddr=127.0.0.1:10030 -device e1000,mac=0c:36:25:76:89:05,netdev=gns3-5 -netdev socket,id=gns3-5,udp=127.0.0.1:10033,localaddr=127.0.0.1:10032 -device e1000,mac=0c:36:25:76:89:06,netdev=gns3-6 -netdev socket,id=gns3-6,udp=127.0.0.1:10035,localaddr=127.0.0.1:10034 -device e1000,mac=0c:36:25:76:89:07,netdev=gns3-7 -netdev socket,id=gns3-7,udp=127.0.0.1:10037,localaddr=127.0.0.1:10036 -display none
2021-01-23 21:06:15 INFO qemu_vm.py:1013 logging to /home/saad/GNS3/projects/network automation/project-files/qemu/1b0875e1-3164-4107-b7fc-8d38acb55ee7/qemu.log
2021-01-23 21:06:15 INFO qemu_vm.py:1021 QEMU VM "R1" started PID=55515
2021-01-23 21:06:15 INFO base_node.py:655 Starting new uBridge hypervisor localhost:34323
2021-01-23 21:06:15 INFO hypervisor.py:169 starting ubridge: ['/usr/bin/ubridge', '-H', 'localhost:34323']
2021-01-23 21:06:15 INFO hypervisor.py:171 logging to /home/saad/GNS3/projects/network automation/project-files/qemu/1b0875e1-3164-4107-b7fc-8d38acb55ee7/ubridge.log
2021-01-23 21:06:15 INFO hypervisor.py:179 ubridge started PID=55521
2021-01-23 21:06:15 INFO base_node.py:658 Hypervisor localhost:34323 has successfully started
2021-01-23 21:06:15 INFO web_log.py:233 ::1 [23/Jan/2021:16:06:15 +0000] "GET /v2/computes HTTP/1.1" 200 1023 "-" "GNS3 QT Client v2.2.17"
2021-01-23 21:06:15 INFO ubridge_hypervisor.py:83 Connected to uBridge hypervisor on localhost:46125 after 0.1051 seconds
2021-01-23 21:06:15 INFO ubridge_hypervisor.py:83 Connected to uBridge hypervisor on localhost:41995 after 0.1024 seconds
2021-01-23 21:06:15 INFO ubridge_hypervisor.py:83 Connected to uBridge hypervisor on localhost:34323 after 0.1015 seconds
2021-01-23 21:06:15 INFO docker_vm.py:499 Docker container 'ubuntu-1' [ubuntu:latest] started listen for telnet on 5005
2021-01-23 21:06:15 INFO web_log.py:233 ::1 [23/Jan/2021:16:06:14 +0000] "POST /v2/compute/projects/eca8779b-e255-4fd3-8992-7e3ae5413625/docker/nodes/910bf8f4-0655-495c-91de-87034fb5bc33/start HTTP/1.1" 204 247 "-" "Python/3.9 aiohttp/3.6.2"
2021-01-23 21:06:15 INFO qemu_vm.py:1147 Connected to QEMU monitor on 127.0.0.1:46611 after 0.0109 seconds
2021-01-23 21:06:15 INFO qemu_vm.py:1212 Execute QEMU monitor command: set_link gns3-3 off
2021-01-23 21:06:15 INFO qemu_vm.py:1212 Execute QEMU monitor command: set_link gns3-4 off
2021-01-23 21:06:15 INFO qemu_vm.py:1212 Execute QEMU monitor command: set_link gns3-5 off
2021-01-23 21:06:15 INFO qemu_vm.py:1212 Execute QEMU monitor command: set_link gns3-6 off
2021-01-23 21:06:15 INFO qemu_vm.py:1212 Execute QEMU monitor command: set_link gns3-7 off
2021-01-23 21:06:15 INFO web_log.py:233 ::1 [23/Jan/2021:16:06:14 +0000] "POST /v2/compute/projects/eca8779b-e255-4fd3-8992-7e3ae5413625/qemu/nodes/9e868b82-c2e7-4ac7-a8ff-95ecf0a71548/start HTTP/1.1" 200 3085 "-" "Python/3.9 aiohttp/3.6.2"
2021-01-23 21:06:15 INFO qemu_vm.py:1147 Connected to QEMU monitor on 127.0.0.1:44821 after 0.0108 seconds
2021-01-23 21:06:15 INFO qemu_vm.py:1212 Execute QEMU monitor command: set_link gns3-0 off
2021-01-23 21:06:15 INFO qemu_vm.py:1212 Execute QEMU monitor command: set_link gns3-2 off
2021-01-23 21:06:15 INFO qemu_vm.py:1212 Execute QEMU monitor command: set_link gns3-3 off
2021-01-23 21:06:15 INFO qemu_vm.py:1212 Execute QEMU monitor command: set_link gns3-4 off
2021-01-23 21:06:15 INFO qemu_vm.py:1212 Execute QEMU monitor command: set_link gns3-5 off
2021-01-23 21:06:15 INFO qemu_vm.py:1212 Execute QEMU monitor command: set_link gns3-6 off
2021-01-23 21:06:15 INFO qemu_vm.py:1212 Execute QEMU monitor command: set_link gns3-7 off
2021-01-23 21:06:15 INFO web_log.py:233 127.0.0.1 [23/Jan/2021:16:06:14 +0000] "POST /v2/compute/projects/eca8779b-e255-4fd3-8992-7e3ae5413625/qemu/nodes/1b0875e1-3164-4107-b7fc-8d38acb55ee7/start HTTP/1.1" 200 3392 "-" "Python/3.9 aiohttp/3.6.2"
2021-01-23 21:06:15 INFO web_log.py:233 ::1 [23/Jan/2021:16:06:14 +0000] "POST /v2/projects/eca8779b-e255-4fd3-8992-7e3ae5413625/nodes/start HTTP/1.1" 204 222 "-" "GNS3 QT Client v2.2.17"
2021-01-23 21:06:51 INFO web_log.py:233 ::1 [23/Jan/2021:16:06:51 +0000] "GET /v2/projects/eca8779b-e255-4fd3-8992-7e3ae5413625/nodes/910bf8f4-0655-495c-91de-87034fb5bc33/files/etc/network/interfaces HTTP/1.1" 200 521 "-" "GNS3 QT Client v2.2.17"
2021-01-23 21:11:04 INFO web_log.py:233 ::1 [23/Jan/2021:16:11:04 +0000] "GET /v2/computes HTTP/1.1" 200 1023 "-" "GNS3 QT Client v2.2.17"
